projects/libexif/exif_loader_fuzzer.cc

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <libexif/exif-loader.h>
+
+
+void content_func(ExifEntry *entry, void *user_data) {
+	char buf[10000];
```

Comment:
Your indentation is not consistent.

Reply:
Thanks Alex, I need to add clang-format to my editor config apparently...